### PR TITLE
Benchmarking report for Ignite vs Postgres

### PR DESCRIPTION
**Pull request by pkommoju**
* Merged ignite and postgres benchmarks into single codebase.
* Added latency plots and raw data.
* Added scripts for running the benchmark and generating plots.
* Added ascii document with description of the benchmark, results and usage.

### CODE REVIEW

File: benchmark/AlcorJdbcBenchmark/.idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" project-jdk-name="11" project-jdk-type="JavaSDK">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
```
Comment (xieus) [inline benchmark/AlcorJdbcBenchmark/.idea/misc.xml]: .idea files are local IDE related and should not be included.

**Review by xieus — COMMENTED** [review]
@pkommoju Pls remove some local files from this PR.

File: benchmark/AlcorJdbcBenchmark/.idea/misc.xml (hunk 1 above)
Comment (pkommoju) [reply benchmark/AlcorJdbcBenchmark/.idea/misc.xml]: Intellij IDE keeps messing it up. I will remove them now.

Comment (pkommoju) [reply benchmark/AlcorJdbcBenchmark/.idea/misc.xml]: Removed and updated the PR.

**Review by xieus — APPROVED** [review]

**Pull request merged**